client/src/components/SignUpModal.vue

```diff
@@ -0,0 +1,331 @@
+<template>
+  <v-row justify="center" items="center" class="width">
+    <v-dialog v-model="dialog" :fullscreen="mobile" max-width="912px">
+      <v-card style="height: 100%">
+        <div v-if="firstPage">
+          <form style="background-color: #ececec">
```

Comment:
pretty sure you can use the style bg-yourcolor in class attribute

I would add this colour into the custom colours config since it will be used in other places